tests/_plugin_loader_helpers.py

```
@@ -0,0 +1,122 @@
+    schedule_job,
+    shutdown_plugins,
+    start_global_scheduler,
+    stop_global_scheduler,
+)
+from tests.constants import TEST_GIT_TIMEOUT
+
+
+class MockPlugin:
+    """Mock plugin class for testing."""
+
+    def __init__(self, name="test_plugin", priority=10):
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)